clio-client/src/test/scala/org/broadinstitute/clio/client/webclient/ClioWebClientSpec.scala

```diff
@@ -143,7 +143,7 @@ class ClioWebClientSpec extends BaseClientSpec with AsyncMockFactory {
   def jsonResponse[A: Encoder](body: A): HttpResponse = {
     val entity = HttpEntity(
       ContentTypes.`application/json`,
-      body.asJson.pretty(defaultPrinter)
+      body.asJson.printWith(defaultPrinter)
```

Comment:
It's a bit surprising that this type of interface change wasn't backward compatible.

Reply:
Agreed